samples/containers/common/build.gradle.kts

```diff
@@ -1,17 +1,27 @@
 plugins {
-  `java-library`
-  kotlin("jvm")
+  kotlin("multiplatform")
 }
 
-dependencies {
-  implementation(project(":workflow-ui:container-common"))
-  implementation(project(":workflow-core"))
+kotlin {
+  jvm { withJava() }
```

Comment:
I get we have equivalent outcomes here, but I do want to review with @rjrjr and @RBusarow whether or not now is the time to make the jump to the KMP DSL for dependencies here, as it will have some learning curve.